java/connector-node/risingwave-connector-service/src/main/resources/mysql.properties

```diff
@@ -17,7 +15,7 @@ database.server.id=${server.id}
 database.connectionTimeZone=+00:00
 # default heartbeat interval 60 seconds
 heartbeat.interval.ms=${debezium.heartbeat.interval.ms:-60000}
-heartbeat.topics.prefix=${debezium.heartbeat.topics.prefix:-RW_CDC_HeartBeat_}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 3 to 20]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Oh you mean the breaking changes in the properties?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/risingwavelabs/risingwave/14151?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->